recipes/ssh_ipykernel_interrupt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ssh_ipykernel_interrupt" %}
+{% set version = "1.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ssh_ipykernel_interrupt-{{ version }}.tar.gz
+  sha256: 15348f2ed9e0a66707bbbd5df042886d23c5a6aa0dcf67f8792c14adef905097
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - nodejs
+    - sysroot_linux-64 ==2.17  # [linux]
+  host:
+    - jupyter-packaging
+    - pip
+    - python
+    - sysroot_linux-64 ==2.17  # [linux]
```

Comment:
The one in build can go once this is a feedstock, this should never be here.
```suggestion
```

Reply:
Got it, I'll try that